tests/base/parser/bad/gold/cmodule-field-id-colon-with.scilla.gold

```diff
@@ -1,11 +1,11 @@
 {
   "errors": [
     {
-      "error_message": "Invalid type in type annotation.\n",
+      "error_message": "Missing or invalid type in type annotation.\n",
       "start_location": {
         "file": "base/parser/bad/cmodule-field-id-colon-with.scilla",
         "line": 9,
-        "column": 21
+        "column": 19
```

Comment:
The line is question is this:
`field truth : bool = True`

Previously the error was pointing to `True`, but now to the `=`. Any idea why?

Reply:
Good question - I honestly don't know. It's probably due to a change in ParserFaults.messages rather than in the parser itself, but that file is black magic to me, and I just fiddle around with it until it works. :-(

I do think it makes more sense to report the error at the `=` rather than at the initialiser, so I don't think it's worth spending time on.